src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -316,6 +324,21 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('password1', options_list=['--password1'], help='Flag indicating if first password should be deleted', action='store_true', required=False)
         c.argument('password2', options_list=['--password2'], help='Flag indicating if second password should be deleted.', action='store_true', required=False)
 
+    with self.argument_context('acr private-endpoint-connection') as c:
+        # to match private_endpoint_connection_command_guideline.md guidelines
+        c.argument('registry_name', options_list=['--registry-name', '-r'], help='The name of the container registry. You can configure the default registry name using `az configure --defaults acr=<registry name>`', completer=get_resource_name_completion_list(REGISTRY_RESOURCE_TYPE), configured_default='acr')
```

Comment:
I understand that we're on a tight schedule, could we support `--id` in another PR? It's ok to have it in the next milestone.

Reply:
ACR commands right now doesn't support `--id`, we will find appropriate time to update across.

Reply:
Sure. I got it. It's same with storage and keyvault since the name of registry is unique across the whole cloud. But we want to provide consistent UX across services for these four commands.